jest.config.js

```diff
@@ -144,7 +144,7 @@ module.exports = {
   // snapshotSerializers: [],
 
   // The test environment that will be used for testing
-  // testEnvironment: "jest-environment-node",
+  testEnvironment: "node",
```

Comment:
saw this in a tutorial; added because it seemed necessary; though would want to confirm if this is actually required